iocore/net/quic/QUICStream.cc

```diff
@@ -332,28 +332,35 @@ QUICStream::recv(const QUICStreamFrame &frame)
     return std::make_unique<QUICConnectionError>(QUICTransErrorCode::FLOW_CONTROL_ERROR);
   }
 
-  QUICConnectionErrorUPtr error = this->_received_stream_frame_buffer.insert(frame);
+  // Make a copy and insert it into the receive buffer because the frame passed is temporal
+  QUICFrame *cloned             = new QUICStreamFrame(frame);
```

Comment:
This (and one for QUICCryptoFrame) dynamic allocation seems like inevitable.

We should probably replace this with QUICStreamFactory::create_stream_frame() and manage a buffer for the instances. However, if we do it, we also need to think about amount of the buffer. In the worst scenario, it would need window size * sizeof(QUICStreamFrame).

I'll create another issue for this.